t/environs/00-start-mirrorcache.sh

```diff
@@ -0,0 +1,20 @@
+#!lib/test-in-container-environs.sh
+# set -ex
+
+# Start MirrorCache in a docker container and forward to host port 80
+
+./environ.sh pg9-system2
+
+./environ.sh mc9 $(pwd)/MirrorCache
+pg9*/status.sh 2 > /dev/null || pg9*/start.sh
+
+pg9*/create.sh db mc_test
+pg9*/sql.sh -f $(pwd)/MirrorCache/sql/schema.sql mc_test
+
+mc9*/configure_db.sh pg9
+
+# listen on all interfaces
+sed -i 's,MOJO_LISTEN=http://127.0.0.1,MOJO_LISTEN=http://*,' mc9*/start.sh
+
+# don't stop container and forward container port 3190 to host port 80
+EXPOSE_PORT=3190 mc9*/start.sh
```

Comment:
I guess EXPOSE_PORT is not needed here because all content of the file is executed inside container.

Reply:
That's right, will remove it